fv3gfs-physics/tests/main/test_integration.py

```diff
@@ -0,0 +1,92 @@
+from dataclasses import fields
+from datetime import timedelta
+
+import numpy as np
+
+import fv3gfs.physics
+import pace.dsl
+import pace.util
+import pace.util.grid
+from pace.stencils.testing import assert_same_temporaries, copy_temporaries
+
+
+def setup_physics():
+    backend = "numpy"
+    layout = (1, 1)
+    physics_config = fv3gfs.physics.PhysicsConfig(
+        dt_atmos=225, hydrostatic=False, npx=13, npy=13, npz=79, nwat=6, do_qa=True
+    )
+    mpi_comm = pace.util.NullComm(
+        rank=0, total_ranks=6 * layout[0] * layout[1], fill_value=0.0
+    )
+    partitioner = pace.util.CubedSpherePartitioner(pace.util.TilePartitioner(layout))
+    communicator = pace.util.CubedSphereCommunicator(mpi_comm, partitioner)
+    sizer = pace.util.SubtileGridSizer.from_tile_params(
+        nx_tile=physics_config.npx - 1,
+        ny_tile=physics_config.npy - 1,
+        nz=physics_config.npz,
+        n_halo=3,
+        extra_dim_lengths={},
+        layout=layout,
+        tile_partitioner=partitioner.tile,
+        tile_rank=communicator.tile.rank,
+    )
+    grid_indexing = pace.dsl.stencil.GridIndexing.from_sizer_and_communicator(
+        sizer=sizer, cube=communicator
+    )
+    quantity_factory = pace.util.QuantityFactory.from_backend(
+        sizer=sizer, backend=backend
+    )
+    dace_config = pace.dsl.DaceConfig(
+        communicator=None,
```

Comment:
Pass the communicator, for parallel physics test so we can run manually orchestration on them